tests/test_rpc_messages.py

```diff
@@ -1,28 +1,27 @@
 import os
-import asyncio
+import subprocess
+import sys
 
 from azure.functions_worker import protos
-from azure.functions_worker import dispatcher
 from azure.functions_worker import testutils
 
 
 class TestGRPC(testutils.AsyncTestCase):
     pre_test_env = os.environ.copy()
 
-    async def _handle_environment_reload_request(self, test_env):
+    def _reset_environ(self):
+        for key, value in self.pre_test_env.items():
+            os.environ[key] = value
```

Comment:
You may also want to unset the dangling os.environ
Since you don't have any so far, this is not a big deal.